### PR TITLE
Add Linters

### PR DESCRIPTION
**Pull request by dims**
Support running  govet/misspell/gofmt/ineffassign/golint using `golangci/golangci-lint`

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -18,3 +18,8 @@ jobs:
       run: sudo modprobe ip_vs
     - name: Test
       run: sudo go test -v ./...
+    - name: Lint
```
Comment (cpuguy83) [inline .github/workflows/test.yml]: Can you add this as a separate job?
This will let it run in parallel.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -18,3 +18,21 @@ jobs:
       run: sudo modprobe ip_vs
     - name: Test
       run: sudo go test -v ./...
+  lint:
+    strategy:
+      matrix:
+        go-versions: [1.12.x, 1.13.x, 1.14.x]
```
Comment (cpuguy83) [inline .github/workflows/test.yml]: Sorry, let's only run lint on 1.14. No need for the matrix here.

Comment (dims) [reply .github/workflows/test.yml]: Done!

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -18,3 +18,17 @@ jobs:
       run: sudo modprobe ip_vs
     - name: Test
       run: sudo go test -v ./...
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{ matrix.go-version }}
```
Comment (cpuguy83) [inline .github/workflows/test.yml]: I think this isn't doing anything. Not sure what CI does in this case (it's an empty string in the logs).

Comment (dims) [reply .github/workflows/test.yml]: whoops fixed.

**Comment by thaJeztah** [comment]
@cpuguy83 PTAL

**Pull request merged**